spec/mailers/contact_mailer_spec.rb

```diff
@@ -34,4 +34,66 @@
       expect(mail.body.encoded).to have_content('Context: http://example.com/catalog/1')
     end
   end
+
+  context "with a harmful content form" do
+    let(:form) do
+      ReportHarmfulLanguageForm.new(valid_attributes)
+    end
+    let(:mail) do
+      described_class.with(form:).harmful_language.deliver_now
+    end
+
+    context "with all fields filled in" do
+      let(:valid_attributes) do
+        {
+          "name" => "Test",
+          "email" => "test@test.org",
+          "message" => "I am concerned about this subject heading",
+          "context" => "http://example.com/catalog/1",
+          "title" => "Example Record"
+        }
+      end
+
+      it "renders the headers" do
+        expect(mail.subject).to eq("[Possible Harmful Language] Example Record")
+        expect(mail.to).to eq(["test-harmful-content@princeton.edu"])
+        expect(mail.from).to eq(["test@test.org"])
+      end
+
+      it "renders the body" do
+        expect(mail.body.encoded).to have_content('Name: Test')
+        expect(mail.body.encoded).to have_content('Email: test@test.org')
+        expect(mail.body.encoded).to have_content('Subject: [Possible Harmful Language] Example Record')
+        expect(mail.body.encoded).to have_content('Comments: I am concerned about this subject heading')
+        expect(mail.body.encoded).to have_content('Context: http://example.com/catalog/1')
+      end
+    end
+
+    context 'with minimal valid attributes' do
+      let(:valid_attributes) do
+        {
+          "name" => "",
+          "email" => "",
+          "message" => "I am concerned about this subject heading",
+          "context" => "http://example.com/catalog/1",
+          "title" => "Example Record"
+        }
+      end
+
+      it "renders the headers" do
+        expect(mail.subject).to eq("[Possible Harmful Language] Example Record")
+        expect(mail.to).to eq(["test-harmful-content@princeton.edu"])
+        expect(mail.from).to eq(["test-harmful-content@princeton.edu"])
+      end
+
+      it "renders the body" do
+        # The behavior for empty Name and Email are TBD at the moment
+        # expect(mail.body.encoded).to have_content('Name: Test')
```

Comment:
Ditto, can we uncomment these expectations?

Reply:
Ditto! Done!